model/src/main/java/org/cloudfoundry/identity/uaa/zone/UserConfig.java

```diff
@@ -20,11 +20,21 @@ public class UserConfig {
 
     private List<String> defaultGroups = DEFAULT_ZONE_GROUPS;
 
+    private Integer maxUsers;
+
```

Comment:
I do not like the `maxUsers` default value being null. What is the real default number that you want to use? `-1`? In any case, I suggest that we make `maxUsers` type to `int` with some default number.